### DISCUSSION TITLE
Github Actions checks missing on historical commits

### DISCUSSION THREAD

**Discussion by bmeeder22** [community/community#43649] [Repositories]
### Select Topic Area

Bug

### Body

Just recently we noticed the checks symbols (Pending, Success, Failed) showing the status of our github actions, stopped showing up in our repository for non currently HEAD commits. It is helpful to see the status of previous commits and easily link to their action runs. This behavior is happening in all of our branches and PR's.

 In `main` only the current HEAD commit shows the status icon, when a new commit is added to the branch, the check icon for that commit disappears and will only appear for the new HEAD commit. 

For PR's you normally see the status of the actions next of the commit hash for every commit, whenever new commits are added to our PR's the status check disappears. The action run can still be seen in our "actions" tab, or if you click directly on the commit hash you can see the icon next to the commit message. 

Initial State: 
<img width="92" alt="Screen Shot 2023-01-07 at 10 25 05 AM" src="https://user-images.githubusercontent.com/13264552/211219724-94992c2d-e6bd-4342-a9ba-ce4a83f040ba.png">

Make a new commit to the PR: 
<img width="136" alt="Screen Shot 2023-01-07 at 10 23 08 AM" src="https://user-images.githubusercontent.com/13264552/211219725-9373bc66-e669-41d9-8015-f9e54a5a9392.png">

Expected behavior: 
<img width="82" alt="Screen Shot 2023-01-08 at 3 29 22 PM" src="https://user-images.githubusercontent.com/13264552/211220033-1b151348-a077-4b99-99c9-8e3050d4b385.png">

**Comment by dkubb** [comment]
My team noticed this too. The bug was introduced in a change sometime between Jan 9th and 10th where the per-commit build status is now being explicitly hidden; even if the data is still available elsewhere.

**Comment by mbj** [comment]
As a data point: some of the "other" commit statuses actually show up if you wait long enough. So keepnig a commit list open for many many minutes may show statuses for a subset of the commits in the list. Used to be instant.

**Comment by Valiheydari** [comment]
***@***.***

در تاریخ دوشنبه ۹ ژانویهٔ ۲۰۲۳،‏ ۱:۰۰ Maydar ***@***.***>
نوشت:
